README.md

```diff
@@ -82,32 +82,32 @@ cluster. A Kubernetes namespace provides the scope for some kind of resources
 in the cluster. Users interacting with one namespace do not see the content in
 another Namespace.
 
-Kubernetes comes with few Namespace resources and leave the administrator to
-create further namespaces in order to create sort of isolated *slices* of the
+Kubernetes comes with few Namespace resources and leaves the administrator to
+create further namespaces to create sort of isolated *slices* of the
 cluster: _Network and Security Policies_, _Resource Quota_, _Limit Ranges_, and
 _RBAC_ are used to enforce isolation among namespaces.
 
 Namespace isolation shines when Kubernetes is used as an enterprise container
-platform, for example, to isolate the production environment from the
+the platform, for example, to isolate the production environment from the
 development and/or to isolate different types of applications.
-Also it works well to isolate applications serving different users when
+Also, it works well to isolate applications serving different users when
 implementing the SaaS business model. 
 
 When implementing a public _CaaS_ platform, the flat namespace structure in
 Kubernetes shows its main limitations. In this model, each new user receives
-their own namespace where to deploy workloads. The user buys a limited amount
+its namespace where to deploy workloads. The user buys a limited amount
```

Comment:
We're using [singular third-person pronoun](https://en.wikipedia.org/wiki/Singular_they) to make the project gender friendly.

Reply:
@YashThakare please, use singular third-person pronouns.